website/components/YouMayLike/index.tsx

```diff
@@ -24,7 +24,11 @@ const YouMayLike = () => {
     const fetchData = async () => {
       setIsLoading(true);
       try {
-        const { data, error } = await supabase.from("restaurants").select();
+        const { data, error } = await supabase
+          .from("restaurants")
+          .select()
+          .range(0, 4);
```

Comment:
I think we should show relevant data instead of first 4 restro?

Reply:
but based on what we should show? as we are not taking location, if so then we can show.

Reply:
Ok, we can do it based on user's data. For now its fine. Let's keep it as it is